### PR TITLE
modify console backend storage plugin to only choose apiserver

### PR DESCRIPTION
**Pull request by BourbonWang**
Frontend: Add select namespace when submitting job. 
Backend: Move apiserver backends to console. If apiserver object backend is unavailable when reading, use mysql if exists.
config: remove uid.

### CODE REVIEW

File: pkg/storage/backends/objects/default/default.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	job, err := d.apiServerBackend.GetJob(ns, name, jobID, kind, region)
+	if err == nil || !d.hasMysql {
+		return job, err
+	}
+	return d.mysqlBackend.GetJob(ns, name, jobID, kind, region)
+}
+
+func (d defaultBackend) ListJobs(query *backends.Query) ([]*dmo.Job, error) {
+	jobs, err := d.apiServerBackend.ListJobs(query)
+	if err == nil || !d.hasMysql {
```
Comment (jian-he) [inline pkg/storage/backends/objects/default/default.go:82]: If the returned jobs are empty,  we can check from mysql

File: pkg/storage/backends/objects/default/default.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	return d.apiServerBackend.StopPod(ns, name, podID)
+}
+
+func (d defaultBackend) SaveJob(job metav1.Object, kind string, specs map[v1.ReplicaType]*v1.ReplicaSpec, jobStatus *v1.JobStatus, region string) error {
+	return d.apiServerBackend.SaveJob(job, kind, specs, jobStatus, region)
+}
+
+func (d defaultBackend) GetJob(ns, name, jobID, kind, region string) (*dmo.Job, error) {
+	job, err := d.apiServerBackend.GetJob(ns, name, jobID, kind, region)
+	if err == nil || !d.hasMysql {
```
Comment (jian-he) [inline pkg/storage/backends/objects/default/default.go:74]: If the returned jobs are empty, we can check from mysql

Comment (BourbonWang) [reply pkg/storage/backends/objects/default/default.go]: Done

**Comment by jian-he** [comment]
lgtm

**Pull request merged**